project-leadership.md

```diff
@@ -46,6 +49,8 @@ One example of success in this area: You are anchoring your team. You’ve been
   <li>Can provide technical leadership on their team, and are able to share their technical vision with team members</li>
 
   <li>Able to anchor small/medium-sized teams with well-defined product priorities and technical direction</li>
+
+  <li>Recognizes when their pair or team would benefit from cutting scope, and builds consensus for it</li>
```

Comment:
How about something like "Recognizes and helps their pair (or small team) feel comfortable cutting scope from a large story or small track"?